app/src/main/res/drawable/rounded_image_background.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    <solid android:color="@color/colorDynamicWidgetPrimary" />
```

Comment:
There are two issues here:
 - this value is for the dynamic color theme and should not be used in other themes
 - this value does not exist on older Android versions (note how it is declared in the `values-v31` directory)